rpm/authselect.spec

```diff
@@ -120,7 +120,14 @@ rm -f $RPM_BUILD_ROOT/%{_libdir}/libauthselect.so
 %{_bindir}/authselect
 %{_mandir}/man8/authselect.8*
 
-%post libs -p /sbin/ldconfig
+%post libs
+/sbin/ldconfig
+# This directory is already created on systems with gdm, but it is not
+# specifically owned by any package. However, it may be missing for example
+# in docker environments thus we need make sure it exists but we should
+# not own it.
+/bin/make -p /etc/dconf/db/distro.d/locks
```

Comment:
Umm this sounds wrong, as system admin I really wouldn't like to have a post script create a random directory on my filesystem.

Reply:
And what would be the correct way?

Reply:
I think creating the directory under $RPM_BUILD_ROOT and then owning it..

Reply:
This does sound wrong as well, because we should not own a directory that is created by other package. Even though the package does not specifically own it. I'm not sure which package creates it.